core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/clouderamanager/ClouderaManagerModificationService.java

```diff
@@ -80,6 +82,53 @@ public void stopCluster(Stack stack) throws CloudbreakException {
 
     @Override
     public int startCluster(Stack stack) throws CloudbreakException {
-        return 0;
+        Cluster cluster = stack.getCluster();
+        ApiClient client = clouderaManagerClientFactory.getClient(stack, cluster);
+        try {
+            startClouderaManager(stack, client);
+            startAgents(stack, client);
+            return startServices(stack, client).getId().intValue();
+        } catch (ApiException e) {
+            LOGGER.info("Couldn't start Cloudera Manager services", e);
+            throw new AmbariOperationFailedException(e.getMessage(), e);
+        }
+    }
+
+    private ApiCommand startServices(Stack stack, ApiClient client) throws ApiException, CloudbreakException {
```

Comment:
Do we have unit test for this functionality? If not, please write.

Reply:
Same for every tests